server/src-lib/Hasura/Backends/Postgres/Connection.hs

```diff
@@ -210,50 +228,123 @@ withTraceContext ctx = \case
           Q.unitQE defaultTxErrorHandler sql () False
      in LTTx $ setTraceContext >> tx
 
-instance Functor (LazyTx e) where
+instance (Functor m) => Functor (LazyTxT e m) where
```

Comment:
Can't that instance be auto-generated with `DeriveFunctor`?

Reply:
Done.